main.tf

```
@@ -33,6 +33,14 @@ resource "github_repository" "default" {
   has_wiki               = var.has_wiki
   is_template            = var.is_template
   visibility             = var.visibility
+  dynamic "template" {
```

```suggestion

  dynamic "template" {
```
Please add a blank line here